node/compose.yaml

```diff
@@ -0,0 +1,27 @@
+version: "3.9"
```

Comment:
docker compose will poorly generalize to N node setup, no?
Or are you planning to generate compose.yaml programmatically?

Reply:
Certainly, this `compose` file was intended for testing purposes, allowing quick deployment of two nodes to verify their communication within containers. With Kubernetes, we could potentially remove this without much issue, as the orchestration of containers using the node image would be handled there. I can remove it and not keep track of it if you consider it, but for now, it serves as a quick test.

Reply:
ack, that's fine